app/components/examplePage1/ExamplePage1Controller.js

```diff
@@ -1,12 +1,37 @@
 (function() {
   'use strict';
   // Controller naming conventions should start with an uppercase letter
-  function ExamplePage1Controller($scope) {
-    $scope.exampleContents1 = 'We are up and running using a required module!';
-    $scope.paragraphText = ' Example Page 1 explicitly references required files';
+  function ExamplePage1Controller($scope, $http) {
+    // Sum the non-itemized data over the summary pages, its in the row labled A,2.
+    var notItemized;
+    $http.get('https://data.oaklandnet.com/resource/rsxe-vvuw.json?$select=sum(amount_a)&$where=elect_date=\'2012-11-06T00:00:00\' and form_type=\'A\' and line_item=\'2\'').
+      success(function(sum) {
+        notItemized = sum[0]['sum_amount_a'];
+      });
+    // Sum by the entity code from schedule A
+    $http.get('https://data.oaklandnet.com/resource/3xq4-ermg.json?$select=sum(tran_amt1), entity_cd&$group=entity_cd&$where=elect_date=\'2012-11-06T00:00:00\'')
```

Comment:
I tried soda-js, but it has basically no documentation.
On Oct 6, 2015 8:31 AM, "Aaron Borden" <notifications@github.com> wrote:

> In app/components/examplePage1/ExamplePage1Controller.js
> <https://github.com/caciviclab/disclosure-frontend-alpha/pull/3#discussion_r41279844>
> :
>
> > @@ -1,12 +1,37 @@
> >  (function() {
> >    'use strict';
> >    // Controller naming conventions should start with an uppercase letter
> > -  function ExamplePage1Controller($scope) {
> > -    $scope.exampleContents1 = 'We are up and running using a required module!';
> > -    $scope.paragraphText = ' Example Page 1 explicitly references required files';
> > +  function ExamplePage1Controller($scope, $http) {
> > +    // Sum the non-itemized data over the summary pages, its in the row labled A,2.
> > +    var notItemized;
> > +    $http.get('https://data.oaklandnet.com/resource/rsxe-vvuw.json?$select=sum(amount_a)&$where=elect_date=\'2012-11-06T00:00:00\' and form_type=\'A\' and line_item=\'2\'').
> > +      success(function(sum) {
> > +        notItemized = sum[0]['sum_amount_a'];
> > +      });
> > +    // Sum by the entity code from schedule A
> > +    $http.get('https://data.oaklandnet.com/resource/3xq4-ermg.json?$select=sum(tran_amt1), entity_cd&$group=entity_cd&$where=elect_date=\'2012-11-06T00:00:00\'')
>
> Great use of $http but you can also pull in soda-js
> <https://github.com/socrata/soda-js> to make the query dsl easier. I can
> help you if you run into issues, but the steps are basically:
>
> Install the dependency
>
> $ npm install --save soda-js
>
> Require (import) the module at the top of the file
>
> var soda = require('soda-js');
>
> and then you can follow the readme from soda-js. Let me know if you have
> any issues.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/caciviclab/disclosure-frontend-alpha/pull/3/files#r41279844>
> .
>